app/src/main/java/com/swiezaki/swiezakimobile/network/error/RetrofitException.kt

```diff
@@ -0,0 +1,11 @@
+package com.swiezaki.swiezakimobile.network.error
+
+sealed class RetrofitException : RuntimeException()
+
+//Todo: parameters may be changed
+data class Http(val response: ErrorResponse) : RetrofitException()
```

Comment:
Well code always may changed, either if you want or not. I do not think that this comment is necessary.